easybuild/easyconfigs/p/Platanus/Platanus-1.2.4-foss-2017a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'MakeCp'
+
+name = 'Platanus'
+version = '1.2.4' 
+
+homepage = 'http://platanus.bio.titech.ac.jp/'
+description = """PLATform for Assembling NUcleotide Sequences"""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['http://platanus.bio.titech.ac.jp/?ddownload=150']
+sources = ['%(name)s_v%(version)s.tar.gz']
+
+# fix Makefile to support newer gcc version
+patches = ['Platanus_new_gcc.patch']
```

Comment:
@ysagon rather than using a patch (which may break with future versions of Platanus), how about something like

```python
buildopts = 'CXXFLAGS="$CXXFLAGS -fpermissive"`
```

This has as extra bonus that EasyBuild is in control of the compiler options.

You probably also need to include `'cstd': 'c++0x'` in `toolchainopts`, and you may also need to explicitly include some of the hardcoded flags like `-Dnullptr=0` and `-lm`